css/fonts.css

```diff
@@ -1,45 +1,9 @@
-@import url('https://fonts.googleapis.com/css2?family=Alegreya+Sans:wght@900&display=swap');
-
-/*h1, h2, h3{
-	font-weight: 900;
-	margin: -5px 0px 0px 5px; Alegreya
-}
-
-*/
-
-
-h2 {
-	color:black;
-	font-size: 15px;
-
-}
-
-
-
-h3 {
-	font-size: 35px;
-	color:black;
-}
-
-
```

Comment:
headings to not seem to work when using a font family; used an internal style sheet on every page instead